packages/core/contracts/insured-bridge/RateModelStore.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Maps rate model objects to L1 token.
+ * @dev This contract is designed to be queried by off-chain relayers that need to compute realized LP fee %'s before
+ * submitting relay transactions to a BridgePool contract. Therefore, this contract does not perform any validation on
+ * the shape of the rate model. The validation step is left as a responsibility to off-chain relayers.
+ */
+contract RateModelStore is Ownable {
```

Comment:
make this multicall so we can get the rate model for a bunch of addresses at the same time.